pkgs/servers/matrix-synapse/matrix-appservice-slack/generate-dependencies.sh

```diff
@@ -1,9 +1,10 @@
-
 #!/usr/bin/env nix-shell
 #! nix-shell -i bash -p nodePackages.node2nix
 
-node2nix \
-  --nodejs-10 \
+ROOT="$(realpath "$(dirname -- "$(readlink -f -- "${BASH_SOURCE[0]}")")"/../../../..)"
+
+$(nix-build $ROOT -A  nodePackages.node2nix --no-out-link)/bin/node2nix \
```

Comment:
If you want to use this, then there is no point to keep the nix-shell shebang above.

Reply:
Good point, resolved.